e2e/test_dataset_manager.py

```diff
@@ -185,8 +226,82 @@ def test_should_create_table(self):
         # then
         self.assertTrue(self.dataset_manager._table_exists('new_table'))
 
+    def test_should_add_labels(self):
+
+        # given
+        self.dataset_manager.create_table('''
+        CREATE TABLE labeled_table (
+            id STRING
+        )''')
+
+        # when
+        test_dataset_id, dataset_manager = create_dataset_manager(
+            config.PROJECT_ID,
+            self.TEST_PARTITION,
+            dataset_name=self.dataset_uuid,
+            internal_tables=self.internal_tables,
+            external_tables=self.external_tables,
+            dataset_labels={'test_label_key': 'test_label_value', 'another_test_label_key': 'another_test_label_value'},
+            tables_labels={'labeled_table': {'labeled_table_key': 'labeled_table_value', 'another_labeled_table_key': 'another_labeled_table_value'}})
+
+        # then
+        self.assertEqual(
+            dataset_manager.client.get_table(test_dataset_id + '.' + 'labeled_table').labels,
+            {'labeled_table_key': 'labeled_table_value', 'another_labeled_table_key': 'another_labeled_table_value'}
+        )
+
+    def test_should_not_add_labels(self):
```

Comment:
I don't get the scenario this test is checking, can you explain?

Reply:
It checks that on the first call there are no labels. It is more like document-test then real testing. I can remove this test. WDYT?

Reply:
remove it then